services/app-web/src/components/FilterAccordion/FilterDateRange/FilterDateRange.tsx

```diff
@@ -0,0 +1,310 @@
+import React, { useRef, forwardRef, useImperativeHandle, useState } from 'react'
+import { FormGroup, Label } from '@trussworks/react-uswds'
+import {
+    DatePicker,
+    DatePickerProps,
+    DatePickerRef,
+} from './_DatePicker/DatePicker'
+import { formatDate, parseDateString } from './_DatePicker/utils'
+import { DEFAULT_EXTERNAL_DATE_FORMAT } from './_DatePicker/constants'
+import classnames from 'classnames'
+import styles from './FilterDateRange.module.scss'
+import { formatUserInputDate } from '../../../formHelpers'
+import { dayjs } from '../../../common-code/dateHelpers/dayjs'
+import { PoliteErrorMessage } from '../../PoliteErrorMessage'
+
+/**
+ * This component uses a modified DatePicker component located in ./_DatePicker. This was done because we needed a ref to
+ * the DatePicker input for clearing input values using the clear filter button on the filter accordion, input
+ * validations, and displaying error messages on each DatePicker input. This component is essentially recreating the
+ * DateRangePicker with the modification to display error messages on the individual DatePicker inputs.
+ *
+ * We will no loner need to a modified DatePicker when:
+ * - DatePicker is updated to surface a ref to the input
+ * - DateRangePicker adds the option to display errors on each DatePicker input
+ */
+
+export type FilterDateRangePropType = {
+    startDateLabel?: string
+    startDateHint?: string
+    startDatePickerProps: Omit<DatePickerProps, 'rangeDate'>
```

Comment:
I would add a comment here for both start date and end date picker props. I think there is basically a required `onChange` for each input in this component. Its intended to be used with an `onChange` that relates to behavior in a parent.  `// expect an onChange to be passed in for updating filter UI and URL params`